Building documentation fails

See https://github.com/DARMA-tasking/magistrate/actions/runs/5191257208/jobs/9358857101

```
Running plantuml with JAVA...
lookup cache used 1188/65536 hits=11561 misses=1205
finished...
WARNING:root:view__serialize_8h.xml: ignoring <programlisting> in <compounddef>
WARNING:root:type__registry_8h.xml: ignoring <programlisting> in <compounddef>
WARNING:root:kokkos__unordered__map__serialize_8h.xml: ignoring <programlisting> in <compounddef>
WARNING:root:checkpoint__api_8h.xml: function parameter description doesn't match parameter names: {'t': ('a valid, constructed <code>T</code> to deserialize into', 'in')}
WARNING:root:checkpoint__api_8h.xml: ignoring <programlisting> in <compounddef>
Traceback (most recent call last):
  File "/build/checkpoint/m.css/documentation/doxygen.py", line 3692, in <module>
    run(doxyfile, templates=os.path.abspath(args.templates), wildcard=args.wildcard, index_pages=args.index_pages, search_merge_subtrees=not args.search_no_subtree_merging, search_add_lookahead_barriers=not args.search_no_lookahead_barriers, search_merge_prefixes=not args.search_no_prefix_merging)
  File "/build/checkpoint/m.css/documentation/doxygen.py", line 3570, in run
    parsed = parse_xml(state, file)
  File "/build/checkpoint/m.css/documentation/doxygen.py", line 2466, in parse_xml
    compound.description, templates, compound.sections, footer_navigation, example_navigation, search_keywords, compound.deprecated, compound.since = parse_toplevel_desc(state, compounddef.find('detaileddescription'))
  File "/build/checkpoint/m.css/documentation/doxygen.py", line 1668, in parse_toplevel_desc
    parsed = parse_desc_internal(state, element)
  File "/build/checkpoint/m.css/documentation/doxygen.py", line 513, in parse_desc_internal
    parsed = parse_desc_internal(state, i)
  File "/build/checkpoint/m.css/documentation/doxygen.py", line 564, in parse_desc_internal
    id = extract_id_hash(state, element)
  File "/build/checkpoint/m.css/documentation/doxygen.py", line 249, in extract_id_hash
    assert id.startswith(state.current_definition_url_base), "ID `%s` does not start with `%s`" % (id, state.current_definition_url_base)
AssertionError: ID `ckpt_learn_ex_traversal_nonintrusive_1ex` does not start with `ckpt_learn_example_polymorphic_nonintrusive`
1
Error: Process completed with exit code 1.
```

--- a/docs/md/checkpoint_learn_example_2.md
+++ b/docs/md/checkpoint_learn_example_2.md
@@ -3,6 +3,6 @@
 The full code for this *checkpoint* example can be found here:
 `examples/checkpoint_example_2.cc`
 
-\subsubsection ex Example source code:
+\subsubsection ex2 Example source code:
 \snippet examples/checkpoint_example_2.cc Serialize custom structure
 

--- a/docs/md/checkpoint_learn_example_2_nonintrusive.md
+++ b/docs/md/checkpoint_learn_example_2_nonintrusive.md
@@ -5,6 +5,6 @@
 The full code for this *checkpoint* example can be found here:
 `examples/checkpoint_example_2_nonintrusive.cc`
 
-\subsubsection ex Example source code:
+\subsubsection ex2 Example source code:
 \snippet examples/checkpoint_example_2_nonintrusive.cc Non-Intrusive Serialize custom structure
 

--- a/docs/md/checkpoint_learn_example_3_nonintrusive.md
+++ b/docs/md/checkpoint_learn_example_3_nonintrusive.md
@@ -5,5 +5,5 @@
 The full code for this *checkpoint* example can be found here:
 `examples/checkpoint_example_3_nonintrusive.cc`
 
-\subsubsection ex Example source code:
+\subsubsection ex3 Example source code:
 \snippet examples/checkpoint_example_3_nonintrusive.cc Non-Intrusive Serialize constraints constructor destructor

--- a/docs/md/checkpoint_learn_example_polymorphic_macro_nonintrusive.md
+++ b/docs/md/checkpoint_learn_example_polymorphic_macro_nonintrusive.md
@@ -1,9 +1,9 @@
-\page ckpt_learn_example_polymorphic_macro_nonintrusive Non-Intrusive Polymorphic Serialization Example w/Macros
+\page ckpt_learn_ex6_polymorphic_macro_nonintrusive Non-Intrusive Polymorphic Serialization Example w/Macros
 
 \attention All Non-Intrusive serialize methods <b>MUST</b> be placed in the namespace of type which they serialize.
 
 The full code for this *checkpoint* example can be found here:
 `examples/checkpoint_example_polymorphic_macro_nonintrusive.cc`
 
-\subsubsection ex Example source code:
+\subsubsection ex6 Example source code:
 \snippet examples/checkpoint_example_polymorphic_macro_nonintrusive.cc Non-Intrusive Serialize polymorphic macro

--- a/docs/md/checkpoint_learn_example_polymorphic_nonintrusive.md
+++ b/docs/md/checkpoint_learn_example_polymorphic_nonintrusive.md
@@ -1,9 +1,9 @@
-\page ckpt_learn_example_polymorphic_nonintrusive Non-Intrusive Polymorphic Serialization Example
+\page ckpt_learn_ex5_polymorphic_nonintrusive Non-Intrusive Polymorphic Serialization Example
 
 \attention All Non-Intrusive serialize methods <b>MUST</b> be placed in the namespace of type which they serialize.
 
 The full code for this *checkpoint* example can be found here:
 `examples/checkpoint_example_polymorphic_nonintrusive.cc`
 
-\subsubsection ex Example source code:
+\subsubsection ex5 Example source code:
 \snippet examples/checkpoint_example_polymorphic_nonintrusive.cc Non-Intrusive Serialize polymorphic

--- a/docs/md/checkpoint_learn_example_to_file.md
+++ b/docs/md/checkpoint_learn_example_to_file.md
@@ -1,4 +1,4 @@
-\page ckpt_learn_ex_tofile Program Example Serialize To File
+\page ckpt_learn_ex4_tofile Program Example Serialize To File
 
 One potential application for this approach is to generate a restart file
 for a simulation.
@@ -6,5 +6,5 @@ for a simulation.
 The full code for this example can be found here:
 `examples/checkpoint_example_to_file.cc`
 
-\subsubsection ex Example source code:
+\subsubsection ex4 Example source code:
 \snippet examples/checkpoint_example_to_file.cc Serialize structure to file

--- a/docs/md/checkpoint_learn_example_to_file_nonintrusive.md
+++ b/docs/md/checkpoint_learn_example_to_file_nonintrusive.md
@@ -1,4 +1,4 @@
-\page ckpt_learn_ex_tofile_nonintrusive Non-Intrusive Program Example Serialize To File
+\page ckpt_learn_ex4_tofile_nonintrusive Non-Intrusive Program Example Serialize To File
 
 \attention All Non-Intrusive serialize methods <b>MUST</b> be placed in the namespace of type which they serialize.
 
@@ -8,5 +8,5 @@ for a simulation.
 The full code for this example can be found here:
 `examples/checkpoint_example_to_file_nonintrusive.cc`
 
-\subsubsection ex Example source code:
+\subsubsection ex4 Example source code:
 \snippet examples/checkpoint_example_to_file_nonintrusive.cc Non-Intrusive Serialize structure to file

--- a/docs/md/checkpoint_learn_example_traversal.md
+++ b/docs/md/checkpoint_learn_example_traversal.md
@@ -1,8 +1,8 @@
-\page ckpt_learn_ex_traversal Program Example Traversal
+\page ckpt_learn_ex7_traversal Program Example Traversal
 
 The full code for this *checkpoint* example can be found here:
 `examples/checkpoint_example_traversal.cc`
 
-\subsubsection ex Example source code:
+\subsubsection ex7 Example source code:
 \snippet examples/checkpoint_example_traversal.cc Serialization with custom traverser
 

--- a/docs/md/checkpoint_learn_example_traversal_nonintrusive.md
+++ b/docs/md/checkpoint_learn_example_traversal_nonintrusive.md
@@ -1,10 +1,10 @@
-\page ckpt_learn_ex_traversal_nonintrusive Non-Intrusive Program Example Traversal
+\page ckpt_learn_ex7_traversal_nonintrusive Non-Intrusive Program Example Traversal
 
 \attention All Non-Intrusive serialize methods <b>MUST</b> be placed in the namespace of type which they serialize.
 
 The full code for this *checkpoint* example can be found here:
 `examples/checkpoint_example_traversal_nonintrusive.cc`
 
-\subsubsection ex Example source code:
+\subsubsection ex7 Example source code:
 \snippet examples/checkpoint_example_traversal_nonintrusive.cc Non-Intrusive Serialization with custom traverser
 

--- a/docs/md/checkpoint_learn_examples.md
+++ b/docs/md/checkpoint_learn_examples.md
@@ -5,10 +5,10 @@
 | checkpoint_example_1 | Serialization of a structure | \subpage ckpt_learn_ex1 |
 | checkpoint_example_2 | Serialization of a custom structure | \subpage ckpt_learn_ex2 |
 | checkpoint_example_3 | Illustrate needs for constructor/destructor | \subpage ckpt_learn_ex3 |
-| checkpoint_example_to_file | Serialize / deserialize structure to file | \subpage ckpt_learn_ex_tofile |
-| checkpoint_example_polymorphic | Serialization of polymorphic class hierarchy | \subpage ckpt_learn_example_polymorphic |
-| checkpoint_example_polymorphic_macro | Serialization of polymorphic class hierarchy with macros | \subpage ckpt_learn_example_polymorphic_macro |
-| checkpoint_example_traversal | Serialization with custom traverser | \subpage ckpt_learn_ex_traversal |
+| checkpoint_example_4_to_file | Serialize / deserialize structure to file | \subpage ckpt_learn_ex4_tofile |
+| checkpoint_example_5_polymorphic | Serialization of polymorphic class hierarchy | \subpage ckpt_learn_ex5_polymorphic |
+| checkpoint_example_6_polymorphic_macro | Serialization of polymorphic class hierarchy with macros | \subpage ckpt_learn_ex6_polymorphic_macro |
+| checkpoint_example_7_traversal | Serialization with custom traverser | \subpage ckpt_learn_ex7_traversal |
 
 &nbsp;
 &nbsp;
@@ -21,7 +21,7 @@ Please do not put serialize methods in global magistrate namespace.
 | checkpoint_example_1 | Serialization of a structure | \subpage ckpt_learn_ex1_nonintrusive |
 | checkpoint_example_2 | Serialization of a custom structure | \subpage ckpt_learn_ex2_nonintrusive |
 | checkpoint_example_3 | Illustrate needs for constructor/destructor | \subpage ckpt_learn_ex3_nonintrusive |
-| checkpoint_example_to_file | Serialize / deserialize structure to file | \subpage ckpt_learn_ex_tofile_nonintrusive |
-| checkpoint_example_polymorphic | Serialization of polymorphic class hierarchy | \subpage ckpt_learn_example_polymorphic_nonintrusive |
-| checkpoint_example_polymorphic_macro | Serialization of polymorphic class hierarchy with macros | \subpage ckpt_learn_example_polymorphic_macro_nonintrusive |
-| checkpoint_example_traversal | Serialization with custom traverser | \subpage ckpt_learn_ex_traversal_nonintrusive |
+| checkpoint_example_4_to_file | Serialize / deserialize structure to file | \subpage ckpt_learn_ex4_tofile_nonintrusive |
+| checkpoint_example_5_polymorphic | Serialization of polymorphic class hierarchy | \subpage ckpt_learn_ex5_polymorphic_nonintrusive |
+| checkpoint_example_6_polymorphic_macro | Serialization of polymorphic class hierarchy with macros | \subpage ckpt_learn_ex6_polymorphic_macro_nonintrusive |
+| checkpoint_example_7_traversal | Serialization with custom traverser | \subpage ckpt_learn_ex7_traversal_nonintrusive |

--- a/docs/shared/checkpoint_learn_example_polymorphic.md
+++ b/docs/shared/checkpoint_learn_example_polymorphic.md
@@ -1,4 +1,4 @@
-\page ckpt_learn_example_polymorphic Polymorphic Serialization Example
+\page ckpt_learn_ex5_polymorphic Polymorphic Serialization Example
 
 The full code for this *checkpoint* example can be found here:
 `examples/checkpoint_example_polymorphic.cc`

--- a/docs/shared/checkpoint_learn_example_polymorphic_macro.md
+++ b/docs/shared/checkpoint_learn_example_polymorphic_macro.md
@@ -1,4 +1,4 @@
-\page ckpt_learn_example_polymorphic_macro Polymorphic Serialization Example w/Macros
+\page ckpt_learn_ex6_polymorphic_macro Polymorphic Serialization Example w/Macros
 
 The full code for this *checkpoint* example can be found here:
 `examples/checkpoint_example_polymorphic_macro.cc`

--- a/docs/shared/checkpoint_learn_serialize.md
+++ b/docs/shared/checkpoint_learn_serialize.md
@@ -88,7 +88,7 @@ hierarchy so it can automatically traverse the hierarchy. Alternatively, the
 user may use the inheritance wrappers `checkpoint::SerializableBase<T>` and
 `checkpoint::SerializableDerived<T, U>` to achieve the same effect.
 
-The example in \ref ckpt_learn_example_polymorphic_macro illustrates the
+The example in \ref ckpt_learn_ex6_polymorphic_macro illustrates the
 approach uses the macros. The example in \ref ckpt_learn_example_polymorphic
 illustrates this approach.
 

